src/lib/services/api-base/api-base.ts

```diff
@@ -204,9 +153,12 @@ export class ApiBase {
 
     try {
       const response = await fetch(`https://${this.apiServer}/v1/${url}`, init);
-      const responseJson = await response.json();
-      this.validate(responseJson);
-      return responseJson;
+
+      if (!response.ok) {
+        throw new Error(`Request failed!`)  
```

Comment:
This message is up for debate. The response object includes information like url, status code etc. Everything that is already present under the "network" tab.